pkg/apiserver/filters/dispatch.go

```diff
@@ -23,9 +24,11 @@ func WithMultipleClusterDispatcher(handler http.Handler, dispatch dispatch.Dispa
 			return
 		}
 
-		if info.Cluster == "" {
+		if info.Cluster == "host-cluster" || info.Cluster == "" {
```

Comment:
empty cluster name implies no dispatching, "host-cluster" is unnecessary

Reply:
Implied cluster info will be completed when parsing request info, dispatching is not needed if cluster is empty or request host-cluster's resources .   https://github.com/kubesphere/kubesphere/pull/1960/files#diff-07d66560df8f80f564e18b05364c82d1R130